hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/management/GetMemberConfigMessageTask.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.client.impl.protocol.task.management;
+
+import com.hazelcast.client.impl.protocol.ClientMessage;
+import com.hazelcast.client.impl.protocol.codec.MCGetMemberConfigCodec;
+import com.hazelcast.client.impl.protocol.codec.MCGetMemberConfigCodec.RequestParameters;
+import com.hazelcast.client.impl.protocol.task.AbstractInvocationMessageTask;
+import com.hazelcast.instance.impl.Node;
+import com.hazelcast.internal.management.ManagementCenterService;
+import com.hazelcast.internal.management.operation.GetMemberConfigOperation;
+import com.hazelcast.internal.nio.Connection;
+import com.hazelcast.spi.impl.operationservice.InvocationBuilder;
+import com.hazelcast.spi.impl.operationservice.Operation;
+
+import java.security.Permission;
+
+public class GetMemberConfigMessageTask extends AbstractInvocationMessageTask<RequestParameters> {
+    public GetMemberConfigMessageTask(ClientMessage clientMessage, Node node, Connection connection) {
+        super(clientMessage, node, connection);
+    }
+
+    @Override
+    protected InvocationBuilder getInvocationBuilder(Operation op) {
+        return nodeEngine.getOperationService().createInvocationBuilder(getServiceName(),
```

Comment:
For the message tasks that run on this member, you don't need the create an operation again. 
Please, use `AbstractCallableMessageTask` instead of `AbstractInvocationMessageTask`
See `ClientStatisticsMessageTask` for example usage.

Reply:
Done 👍 I have left `PromoteLiteMemberOperation` intact as I needed it for executing asynchronously. The ones that already have a `LocalOperation` are left as well. They  can be converted to this once we get to clean up the old `ConsoleRequest` implementations we need for the old MC operations.